### DISCUSSION TITLE
Phones & Contacting the Police

### DISCUSSION THREAD

**Discussion by jasper-one** [EveronLife/EveronLife#89] [Ideas]
I've been thinking about how phones can/should be implemented and found the following in-game assets ( the mobile phone would need to be implemented) and how they could be used. 

**Phone Booths**
These are available already at 17 locations across the map and could be free to use to do things like contact Police, Ambulance, other phone booths, office/home phones, and mobile phones.
![image](https://user-images.githubusercontent.com/106525617/172033071-84aaf611-bb42-4bea-9d10-2f8cd3a6a03a.png)

**Phones in Offices & Homes**
These are common in virtually every building, so player homes could all have their own personal phones. Again, they would be free to use as long as you can access them.  
![image](https://user-images.githubusercontent.com/106525617/172033050-5be8290f-c2d6-4e89-8639-f10a48cfd6db.png)

**Time Period Accurate Phones**
Phones were very expensive at the time, adjusted for inflation they were about $10,420. So owning your own phone could be something you have to save up for in the early game, rather than just something you spawn with like in Altis life. Certain players might even play without one, bringing interesting RP situations like having to run to the closest phone booth to get help because you don't have a phone / it got stolen.  
![image](https://user-images.githubusercontent.com/106525617/172033090-134439ca-2ae6-4273-ae69-2cf101904c86.png)

I wanted to share this idea and get people's thoughts. I really think it would provide great immersion and RP experiences. If someone would like to team up to implement this, give me a bell, I'd love to help where I can (modeling / currently learning enfusion scripting).

**Comment by borknbeans** [comment]
I'd love to try and help with the scripting side of things! Ill message you on discord!

**Comment by SaxW** [comment]
@ebork11 I just built this model if we're keen to use it. 
![image](https://user-images.githubusercontent.com/53883603/172347387-525f700d-8141-4b68-8a4a-b1906328a942.png)
![image](https://user-images.githubusercontent.com/53883603/172347502-04df9590-cc34-4c32-a017-1d4068ed9415.png)

**Reply by PackJC** [reply]
That model looks great.  I am imagining all of the fun if this concept becomes real. Phone trolling would be unbelievable hilarious.

**Reply by borknbeans** [reply]
Id love to try to implement it if you want to send it over!

**Comment by Arkensor** [comment]
I am a big fan of the concept of using that different kinds of phones. I think accessibility and cost could be distinctive factors for this.
